chanzuckerberg/idseq-web
# surface informative error if too many read pairs are broken

Goes with https://github.com/chanzuckerberg/idseq-dag/pull/171.

# Tests
- Run a samples whose read pairs don't match up and see the pipeline fail gracefully with an informative error.
- Run a normal test sample and see it succeed.
